src/components/dashboard/Dashboard.jsx

```diff
@@ -42,29 +52,39 @@ class Dashboard extends React.Component {
         });
     }
 
-    async setDashboardStyling(iframe) {
+    async setDashboardStyling(iframe, dataSetId) {
         const iframeDocument = iframe.contentWindow.document;
 
         await this.waitforDashboardToLoad(iframeDocument);
         const iFrameRoot = iframeDocument.querySelector("#root");
-        iFrameRoot.style.marginTop = "-110px";
+
         const iFrameWrapper = iframeDocument.querySelector(".app-wrapper");
         iFrameWrapper.removeChild(iFrameWrapper.firstChild).remove();
-        iFrameWrapper.removeChild(iFrameWrapper.firstChild).remove();
+        if (dataSetId) {
+            iFrameRoot.style.marginTop = "-110px";
+            iFrameWrapper.removeChild(iFrameWrapper.firstChild).remove();
```

Comment:
There is a problem with the action [Edit], React fails to finds the bar with the dashboards, which it uses to show on edit mode the save/cancel changes: `Uncaught DOMException: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node`.

I tried to hide instead the bar instead of removing it, but then the edit-mode actions are hidden are the user cannot exit the edit.

Maybe the easiest solution is to remove the edit button.